minishift/README.md

```diff
@@ -0,0 +1,140 @@
+
+Make sure to verify that your console is configured to reuse the Minishift Docker daemon by running `docker ps` command. You should be able to see running containers for origin.
+If not then follow [this](https://docs.openshift.org/latest/minishift/using/docker-daemon.html#docker-daemon-overview) guide to configure it.
+
+#### Create a Project
+Let's create a new project by executing following make target.
+```bash
+make init-project
+```
+
+This will create a new project with name `fabric8-services` with `developer:developer` account and switch to it. Make sure to verify that using `oc project`.
+
+#### Auth Service
+##### Deploying Auth
+
+To deploy auth service, we have following make target which will deploy required secrets, postgres DB and create routes for you.
+```
+make deploy-auth
+```
+
+Look for running pods using `oc get pods`. You should be able to see two pods(auth-*, db-auth-*). First time it will take some time as it has download required container images.
+
+##### Check auth service status
+You can get auth route by using `oc get routes`. It should be in format `auth-fabric8-services.${minishift ip}.nip.io`
+You can check status by hitting this in browser `http://auth-fabric8-services.${minishift ip}.nip.io/api/status`(e.g. `http://auth-fabric8-services.192.168.42.177.nip.io/api/status`).
+
+##### Connecting to Postgres DB
+If you wish to access the Postgres database, it is available on the same host but on port 31001.  Use the following command to connect with the Postgres client:
+
+```bash
+PGPASSWORD=mysecretpassword psql -h minishift.local -U postgres -d postgres -p 31001
+```
+
+#### Cluster Service
+##### Deploying Cluster
+
+To deploy cluster service, we have following make target which will deploy required secrets, config map, postgres DB and create routes for you.
+```
+make deploy-cluster
+```
+
+Look for running pods using `oc get pods`. You should be able to see two pods(f8cluster-*, db-f8cluster-*). First time it will take some time as it has download required container images.
+
+##### Check Cluster service status
+You can get cluster route by using `oc get routes`. It should be in format `f8cluster-fabric8-services.${minishift ip}.nip.io`
+You can check status by hitting this in browser `http://f8cluster-fabric8-services.${minishift ip}.nip.io/api/status`(e.g. `http://f8cluster-fabric8-services.192.168.42.177.nip.io/api/status`).
+
+##### Connecting to Postgres DB
+If you wish to access the Postgres database, it is available on the same host but on port 31002.  Use the following command to connect with the Postgres client:
+
+```bash
+PGPASSWORD=mysecretpassword psql -h minishift.local -U postgres -d postgres -p 31002
+```
+
+#### Deploying Auth, Cluster together
+To deploy `auth`, `f8cluster` together we have following target:
+```bash
+make deploy-all
```

Comment:
Shouldn't we start with some quick start instructions in the beginning of the readme? And add all details and additional targets after that?

WDYT about having targets similar to what we have in Auth - https://github.com/fabric8-services/fabric8-auth/blob/master/minishift/README.md
For example `make dev-openshift` to start everything. We can keep your targets too.